pyupgrade.py

```diff
@@ -2477,7 +2505,7 @@ def _fix_file(filename: str, args: argparse.Namespace) -> int:
     if not args.keep_percent_format:
         contents_text = _fix_percent_format(contents_text)
     if args.min_version >= (3,):
-        contents_text = _fix_py3_plus(contents_text)
+        contents_text = _fix_py3_plus(contents_text, args.min_version)
```

Comment:
this one might be simple enough to do with `_fix_tokens` -- then you won't need to rewrite every call to `_fix_py3_plus`

Reply:
If I do that would I just search for `lru_cache()`? If so, how would I avoid rewriting `from not_functools import lru_cache; @lru_cache()`?

Reply:
I could make `min_version` an optional arg to avoid rewriting `_fix_py3_plus` calls